modules/gapi/test/common/gapi_video_tests_inl.hpp

```diff
@@ -88,6 +89,34 @@ TEST_P(BuildPyr_CalcOptFlow_PipelineTest, AccuracyTest)
     compareOutputsOptFlow(outOCV, outGAPI);
 }
 
+TEST_P(BackgroundSubtractorMOG2Test, AccuracyTest)
+{
+    initTestDataPath();
+
+    // G-API graph declaration
+    cv::GMat in;
+    cv::GMat out = cv::gapi::video::GBackgroundSubtractorMOG2::on(in, -1);
+    // Preserving 'in' in output to have possibility to compare with OpenCV reference
+    cv::GComputation c(cv::GIn(in), cv::GOut(cv::gapi::copy(in), out));
+
+    // G-API compilation of graph for streaming mode
+    const auto pkg = cv::gapi::kernels<GCPUBackgroundSubtractorMOG2>();
+    auto gapiBackSub = c.compileStreaming(cv::compile_args(pkg, cv::gapi::video::BackgroundSubtractorParams{500, 16, true}));
+
+    //auto gapiBackSub = c.compileStreaming(getCompileArgs());
```

Comment:
I think this way is preferable to pass the kernel package. Is it necessary to manually pass `GCPUBackgroundSubtractorMOG2` to `GComputation`?

Reply:
I need to discuss.

Reply:
/cc @dmatveev

Reply:
Done.